### PR TITLE
This reverts commit b6a01e5.

### PR DESCRIPTION
**Pull request by Decodetalkers**
wrong fix

This reverts commit b6a01e5dfad16e17446090c0281edab0a1a90391.

### CODE REVIEW

**Review by zsien — DISMISSED** [review]

File: keybinding/manager.go [hunk 1]
```diff
@@ -554,7 +554,7 @@ func (m *Manager) listenGlobalAccel(sessionBus *dbus.Conn) error {
 								logger.Warning(err)
 							}
 						} else {
-							err := m.execCmd(m.shortcutCmd, true)
+							err := m.execCmd("",m.shortcutCmd, true)
```
Comment (kegechen) [inline keybinding/manager.go:557]: 空格， 下面一样

Comment (kegechen) [inline keybinding/manager.go:557]:
```suggestion
							err := m.execCmd("", m.shortcutCmd, true)
```

**Review by zsien — APPROVED** [review]

**Pull request merged**